theme/common.less

```diff
@@ -211,10 +211,6 @@ div.simframe > iframe {
     .ui.item.link:focus {
         top:20em;
     }
-
-    .ui.item.link:hover {
-        top:20em;
```

Comment:
The hidden menu is in fact visible (otherwise we could not tab on it) but has a negative top value, so it's hidden from the user on the screen. By removing this CSS, we prevent to keep the hidden menu showed when it loose the focus and that the mouse is hover the menu bar. It happened to me a lot of time in fact. It's better like this.